docs/pages/database-access/guides/aws-opensearch.mdx

```diff
@@ -200,42 +200,34 @@ your terminal, and manually adjust `/etc/teleport.yaml`.
 
 Generate a configuration file at `/etc/teleport.yaml` for the Database Service:
 
+<ScopedBlock scope={["oss", "enterprise"]}>
```

Comment:
I'm trying to deprecate the `ScopedBlock` component since the scope switcher is confusing for users, and `ScopedBlocks` hide content unless users figure out how the scope switcher works.

What would you think of using `Tabs` here?

Reply:
That's fine. Is there an example to follow?

Reply:
You would replace each `ScopedBlock` with a `TabItem` and enclose all of the `TabItem`s in a `Tabs` tag. Each `TabItem` would have a scope-specific label, e.g., "Self-Hosted" and "Teleport Cloud".

Reply:
Thanks, please see updates.